client/src/components/Search/SearchBox.js

```diff
@@ -44,30 +45,31 @@ class SearchBox extends Component {
   render() {
     const { id, searchText, hideable, onChange, expanded, showFilters, placeholder } = this.props;
 
-    // Build classes
-    const searchClasses = ['search-box', 'search-box__active'];
-    const advancedButtonClasses = [
-      'btn', 'btn-secondary', 'btn--icon-md', 'btn--no-text',
-      'font-icon-caret-down-two', 'search-box__filter-trigger',
-    ];
-    if (!expanded) {
-      advancedButtonClasses.push('collapsed');
-    }
-
-    // Decide if we display the X button
-    searchClasses.push(hideable ? 'search-box__hideable' : 'search-box__not-hideable');
-    searchClasses.push(this.state.hasFocus ? 'search-box__has-focus' : 'search-box__has-not-focus');
-    searchClasses.push(showFilters ? 'search-box__has-filters' : 'search-box__has-not-filters');
-
+    const searchClasses = classNames('search-box', {
+      'search-box__hideable': hideable,
+      'search-box__not-hideable': !hideable,
+      'search-box__has-focus': this.state.hasFocus,
+      'search-box__has-not-focus': !this.state.hasFocus,
+      'search-box__has-filters': showFilters,
+      'search-box__has-not-filters': !showFilters
+    });
+
+    const advancedButtonClasses = classNames(
+      'btn--icon-md',
+      'btn--no-text',
+      'font-icon-caret-down-two',
+      'search-box__filter-trigger',
+      { collapsed: !expanded }
+    );
 
     return (
-      <div className={searchClasses.join(' ')}>
+      <div className={searchClasses}>
         <div className="search-box__group">
           <Label for={id} id={`${id}_label`} hidden>{i18n._t('Admin.SEARCH', 'Search')}</Label>
           <input
             aria-labelledby={`${id}_label`}
             type="text"
-            name="name"
+            name={this.props.name}
```

Comment:
This should probably be a part of the destructuring of `this.props` above

Reply:
Done.